src/App.css

```
@@ -0,0 +1,328 @@
+  margin-bottom: 20px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.logo-text {
+  font-size: 24px;
+  font-family: 'GothamBold', sans-serif;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "GothamBold" (font-family-name-quotes)

(font-family-name-quotes)